package-lock.json

```diff
@@ -16,7 +16,7 @@
       "integrity": "sha1-63d99gEXI6OxTopywIBcjoZ0a9I=",
       "dev": true,
       "requires": {
-        "mime-types": "2.1.18",
+        "mime-types": "~2.1.18",
```

Comment:
I think the problem lays in your tooling, because when I execute `npm i`, it does _not_ add these symbols. There is no reason why your PR should update this file.